test/Dialect/Handshake/func.mlir

```diff
@@ -0,0 +1,21 @@
+// RUN: circt-opt -split-input-file --handshake-materialize-forks-sinks %s | FileCheck %s
```

Comment:
Why the `-handshake-materialize-forks-sinks` here?

Reply:
Whups, I forgot to remove that.